exec/vm.go

```diff
@@ -47,6 +47,7 @@ type context struct {
 	stack   []uint64
 	locals  []uint64
 	code    []byte
+	asm     *[]asmBlock
```

Comment:
could you explain why the extra indirection?

Reply:
Nope! fixed.